docker-compose.yaml

```diff
@@ -21,7 +21,7 @@ services:
     working_dir: /microsoft/modsecurity
     entrypoint: 
       - ./configure
-      - CXXFLAGS=-std=c++11
+      - CXXFLAGS=-std=c++14

```

Comment:
why will we move compiler version,  what kind of benefit？

Reply:
It allows us to use `std::optional` and potentially some other useful language features.
`std::optional` is used in the latest revision of the patchset.